Flaky test: test_projection_rebuild_with_required_columns/test.py::test_projection_rebuild_uses_only_required_columns

_Important: This issue was automatically generated and is used by CI for matching failures. DO NOT modify the body content. DO NOT remove labels._

Test name: test_projection_rebuild_with_required_columns/test.py::test_projection_rebuild_uses_only_required_columns
Failure reason: AssertionError: assert '0\n' == '2\n'
CI report: [Integration tests (arm_binary, distributed plan, 4/4)](https://s3.amazonaws.com/clickhouse-test-reports/json.html?PR=92888&sha=c49c38006b7426a6ea169f9b6970eb5dd86de5e1&name_0=PR&name_1=Integration%20tests%20%28arm_binary%2C%20distributed%20plan%2C%204%2F4%29)

CIDB statistics: [cidb](https://play.clickhouse.com/play?user=play&run=1#V0lUSAogICAgOTAgQVMgaW50ZXJ2YWxfZGF5cwpTRUxFQ1QKICAgIHRvU3RhcnRPZkRheShjaGVja19zdGFydF90aW1lKSBBUyBkYXksCiAgICBjb3VudCgpIEFTIGZhaWx1cmVzLAogICAgZ3JvdXBVbmlxQXJyYXkocHVsbF9yZXF1ZXN0X251bWJlcikgQVMgcHJzLAogICAgYW55KHJlcG9ydF91cmwpIEFTIHJlcG9ydF91cmwKRlJPTSBjaGVja3MKV0hFUkUgKG5vdygpIC0gdG9JbnRlcnZhbERheShpbnRlcnZhbF9kYXlzKSkgPD0gY2hlY2tfc3RhcnRfdGltZQogICAgQU5EIHRlc3RfbmFtZSA9ICd0ZXN0X3Byb2plY3Rpb25fcmVidWlsZF93aXRoX3JlcXVpcmVkX2NvbHVtbnMvdGVzdC5weTo6dGVzdF9wcm9qZWN0aW9uX3JlYnVpbGRfdXNlc19vbmx5X3JlcXVpcmVkX2NvbHVtbnMnCiAgICAtLSBBTkQgY2hlY2tfbmFtZSA9ICdJbnRlZ3JhdGlvbiB0ZXN0cyAoYXJtX2JpbmFyeSwgZGlzdHJpYnV0ZWQgcGxhbiwgNC80KScKICAgIEFORCB0ZXN0X3N0YXR1cyBJTiAoJ0ZBSUwnLCAnRVJST1InKQogICAgQU5EICgocHVsbF9yZXF1ZXN0X251bWJlciA9IDAgQU5EIGhlYWRfcmVmID0gJ0F6dXJlX2ludmFsaWRfYmxvY2tfbGlzdCcpIE9SIChwdWxsX3JlcXVlc3RfbnVtYmVyICE9IDAgQU5EIGJhc2VfcmVmID0gJ21hc3RlcicpKQpHUk9VUCBCWSBkYXkKT1JERVIgQlkgZGF5IERFU0MK)

Test output:
```
File: test_projection_rebuild_with_required_columns/test.py:29 - in test_projection_rebuild_uses_only_required_columns
    assert (cnt == '2\n')
E   AssertionError: assert '0\n' == '2\n'
E     
E     - 2
E     + 0
```

--- a/tests/integration/test_projection_rebuild_with_required_columns/test.py
+++ b/tests/integration/test_projection_rebuild_with_required_columns/test.py
@@ -19,12 +19,13 @@ def started_cluster():
 
 
 def test_projection_rebuild_uses_only_required_columns(started_cluster):
+    node1.query("drop table if exists tab")
     node1.query("create table tab (x UInt64, y UInt64, data String codec(NONE), v UInt8, projection p (select _part_offset order by y)) engine = ReplacingMergeTree(v) order by x settings allow_part_offset_column_in_projections=1, deduplicate_merge_projection_mode='rebuild';")
     node1.query("insert into tab select number, number, rightPad('', 100, 'a'), 0 from numbers(30000);")
     node1.query("optimize table tab final settings mutations_sync=2, alter_sync=2;")
     node1.query("system flush logs;")
 
     uuid = node1.query("select uuid from system.tables where table = 'tab';").strip()
-    cnt = node1.query("select count() from system.text_log where query_id = '{}::all_1_1_2' and message like '%Reading%from part p_%from the beginning of the part%'".format(uuid))
+    cnt = node1.query("select count() from system.text_log where query_id like '{}::all_%_2' and message like '%Reading%from part p_%from the beginning of the part%'".format(uuid))
     assert (cnt == '2\n')
 

